hibernate-envers/src/main/java/org/hibernate/envers/boot/internal/EnversServiceImpl.java

```diff
@@ -182,15 +182,8 @@ private static AuditStrategy initializeAuditStrategy(
 			);
 		}
 
-		if ( strategy instanceof ValidityAuditStrategy ) {
-			// further initialization required
-			final Getter revisionTimestampGetter = ReflectionTools.getGetter(
-					revisionInfoClass,
-					revisionInfoTimestampData,
-					serviceRegistry
-			);
-			( (ValidityAuditStrategy) strategy ).setRevisionTimestampGetter( revisionTimestampGetter );
-		}
+		// Strategy-specific initialization
+		strategy.initializeAdditional(revisionInfoClass, revisionInfoTimestampData, serviceRegistry);
```

Comment:
Need some padding in the parentheses (shoud be `strategy.initializeAdditional( revisionInfoClass, revisionInfoTimestampData, serviceRegistry );`).

Not very excited about the name of the method :).